test/copy-to.js

```diff
@@ -71,6 +71,19 @@ describe('copy-to', () => {
       })
     })
 
+    it('correcly handle error in sql request', (done) => {
+      assertCopyToResult('COPY --wrong-- TO STDOUT', (err, chunks, result, stream) => {
+        assert.notEqual(err, null)
+        const expectedMessage = 'syntax error at end of input'
+        assert.notEqual(
+          err.toString().indexOf(expectedMessage),
+          -1,
+          'Error message should mention reason for query failure.'
+        )
+        done()
+      })
+    })
+
```

Comment:
In this file, there's a `hasCompleted` guard in `assertCopyToResult()` that can now be removed since  both 'error' and 'end' don't happen anymore. 

(Github doesn't allow commenting on unchanged lines.)